package-lock.json

```diff
@@ -27713,6 +27728,17 @@
           "integrity": "sha512-dOy+3AuW3a2wNbZHIuMZpTcgjGuLU/uBL/ubcZF9OXbDo8ff4O8yVp5Bf0efS8uEoYo5q4Fx7dY9OgQGXgAsQA==",
           "dev": true
         },
+        "fs-extra": {
```

Comment:
How careful are we about reviewing package-lock changes? I'm not going to pretend to deeply understand what changes to expect to see to it -- I'm usually wrong :)  . e.g. I see fs-extra is added here, and is a dependency of the cypress dependency you upgraded. It was a dependency of the previous cypress version too although it was a different version. I would have expected to see a version bump change to fs-extra instead of an addition.